tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1018,18 +1015,21 @@ Then(
 )
 
 When(
-  'the user opens the actions sidebar panel of file/folder {string} on the webUI',
-  async function(resource) {
-    await client.page.FilesPageElement.filesRow().openFileActionsMenu(resource)
+  /^the user opens the actions sidebar panel of (file|folder) "([^"]*)" on the webUI$/,
```

Comment:
O.o

Reply:
just an improvement to be strict with the `file` or `folder` element type. ;-)